contributing_to_docs/doc_guidelines.adoc

```diff
@@ -943,7 +943,7 @@ To indicate that a feature is deprecated, include the `modules/deprecated-featur
 For more information on how this is applied, see link:https://github.com/openshift/openshift-docs/pull/31776/files[this example PR].
 
 == Verification of your content
-All documentation changes must be verified by a QE team associate before merging. This includes executing all "Procedure" changes and confirming expected results. There are exceptions for typo-level changes, formatting-only changes, and other negotiated documentation sets and distributions.
+All documentation changes must be verified by a QE team associate before merging. This includes executing all "Procedure" changes, confirming expected results, and conceptual updates. There are exceptions for typo-level changes, formatting-only changes, and other negotiated documentation sets and distributions.
```

Comment:
```suggestion
All documentation changes that update or add technical content must be verified by a QE team associate before merging. This QE verification process includes executing all procedures, confirming expected results, and confirming the accuracy of conceptual and reference content. The only exceptions are for typo-level changes, formatting-only changes, and other negotiated documentation sets and distributions.
```

Reply:
Not sure if this falls under negotiated documentation sets, but not sure if we should mention refactoring existing content when no changes are being made to the content.

Reply:
Ah. Please disregard. I believe my concern is covered by "changes that update or add technical content"

Reply:
For the exceptions list, I was wondering about adding links?

Reply:
@skrthomas, do you have an example? I think that in most cases, it would be worth getting QE to take a look.

Reply:
@kalexand-rh in this [PR](https://github.com/openshift/openshift-docs/pull/69770), I initially thought I didn't need to have it QE approved. It adds a link to the Loki docs. The Loki content was developed and QE approved already in the Loki docs team, and the docs bug for my component was basically a request for Loki information, which doesn't technically belong in Network Observability even though the Operator requires Loki. So once the content was developed and merged by the Loki writers, I added a link in the NetObserv docs. I ended up getting QE to take a look anyways because actually this whole thread about what needs QE approvals came up the same days I was seeking review 😅 but initially I thought that the content was approved by the Loki team and adding a link in our docs wouldn't be something QE should validate. But I guess, they _could_ validate that the steps in the link are the same as what the bug reporter is asking.